### DISCUSSION TITLE
Setup Without Package Manager

### DISCUSSION THREAD

**Discussion by wsricardo** [nvim-tree/nvim-tree.lua#2894] [Q&A]
Hi,

I trying setup nvim-tree without and with packer for install plugin but dont get success. I have dirs _plugin_ and _lua_ with files from nvim-tree inside dir ~/local/nvim . In doc, section *Quickstart* have code for init.lua, where i must insert init.lua? 
You don't quite understand the installation procedure.

Thanks. Regards.

**Comment by alex-courtis** [comment]
Do you have an existing `init.lua` or `init.vim`?
Are you new to neovim?
Do you have an existing setup with other plugins?
Are you currently using a plugin manager successfully?

You could try ["Clean Room" Replication](https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting#clean-room-replication) to get started.
